### ISSUE TITLE
Refactor InstructorFeedbackResultsPage jsp files

### ISSUE THREAD

**Issue by kanghj** [TEAMMATES/repo#2779]
After #2741 is merged in, the jsp files should be refactored and properly indented, as the bulk of the entire file are not correctly indented now.

**Comment by wkurniawan07** [comment]
No longer relevant I believe, it's all tags now.
